AGENTS.md

```
@@ -64,6 +64,68 @@ bash .claude/hooks/skills-setup.sh
+4. **`/speckit.tasks`** - 実行可能なタスクリストを生成
+5. **`/speckit.implement`** - すべてのタスクを実行して実装
+
+### オプションコマンド
+
+- **`/speckit.constitution`** - プロジェクトの基本原則を作成・更新
+- **`/speckit.analyze`** - アーティファクト間の一貫性を分析
+- **`/speckit.checklist`** - 品質チェックリストを生成
+
+### ディレクトリ構造
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)